### ISSUE TITLE
Support for websockets!

### ISSUE THREAD

**Issue by Shujito** [decebals/pippo#275]
Hello, is there a plan on supporting websockets?
Or a way to work it around? e.g. extract the jetty server and configure it outside.
Thanks for reading

**Comment by decebals** [comment]
I will think for a possible and simple approach. The idea is that Pippo comes with support for many servlets containers (undertow, tomcat, ...) and Jetty is one of these so we must to find a generic approach. Any idea is welcome.

**Comment by decebals** [comment]
Last night, I started working on this feature and the implementation is almost done for Jetty. I created a tiny layer to abstract some basic concepts (WebSocketFilter extends PippoFilter, WebSocketProcessor/Handler, WebSocketConnection, ...). I will propose an API and an implementation for Jetty with a PullRequest.
All I want to do related to this feature (and coming features) is to preserve the simple, non intrusive and configurable character of Pippo, so to achieve this I must to continuously refine the code and this task takes time.
